import-service/pubsub.tf

```diff
@@ -0,0 +1,51 @@
+resource "google_pubsub_topic" "import-service-pubsub-topic" {
+  project = module.import-service-project.project_name
+  name = "import-service-notify"
+}
+
+resource "random_uuid" "pubsub-secret-token" {}
+
+resource "google_pubsub_subscription" "import-service-pubsub-subscription" {
+  project = module.import-service-project.project_name
+  name  = "import-service-notify-push"
+  topic = google_pubsub_topic.import-service-pubsub-topic.name
+
+  ack_deadline_seconds = 600
+
+  expiration_policy {
+    ttl = ""
+  }
```

Comment:
same here ... should subscription include ENV?

Reply:
same response